### PR TITLE
Fix bug not adding to last index, issue #144

### LINKED ISSUE #144: [Bug] Adding a show places the new show to a random index instead of the last index
Adding to show sometimes won't add to the last index

For example: adding "mix" should add to the number 5 but it was added to number 2.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by JonathanHoshi — APPROVED** [review]
LGTM!

File: src/main/java/seedu/trackermon/model/Model.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
     Predicate<Show> PREDICATE_SHOW_ALL_SHOWS = unused -> true;
 
     /** {@code Comparator} that always evaluate to 0 */
```
Comment (Ardentsoul) [inline src/main/java/seedu/trackermon/model/Model.java:18]: Shouldnt this be `-1` instead of `0`

**Review by Ardentsoul — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/trackermon/model/Model.java (hunk 1 above)
Comment (lowkaiwei98) [reply src/main/java/seedu/trackermon/model/Model.java]: Updated!

**Review by Ardentsoul — APPROVED** [review]
LGTM!

**Pull request merged**